lib/phlex/elements.rb

```diff
@@ -38,19 +38,38 @@ def register_element(method_name, tag: nil)
 		class_eval(<<-RUBY, __FILE__, __LINE__ + 1)
 			# frozen_string_literal: true
 
-			def #{method_name}(**attributes, &block)
+			def #{method_name}(object = nil, **attributes, &block)
 				target = @_context.target
 
+				if object
+			    attributes_method = "#{method_name}_attributes"
```

Comment:
Yup! Any thoughts on including attributes in these method signatures? I think it might make a bigger mess if it's included, but it could be handy in the pattern matchers.

Reply:
I’m not sure. It might be handy to send them. They can always be ignored. 🤔

Reply:
I pushed changes that allow Phlex to check for `html_content(tag)` and `html_attributes(tag)`.

I decided to leave attributes out because I think it could make the API confusing. I think what we want is "any HTML attributes passed into the Phlex HTML entity tag method with the object shall overwrite whatever the object emits". That makes it good for one-off modifications. 

Passing attributes into the object could lead people to believe that the could over-write the attributes passed into the Phlex HTML entity tag method.

TBH I really don't know what the answer is here, but I think one will appear as this gets used more. It's easier to add that later once the problem is better understood. I suspect as I continue implementing more, I'll run into actual use cases where I need the attributes ... or I don't.

Reply:
Demo app now implements against poylmprhic tags at https://github.com//rocketshipio/oxidizer-demo/blob/0010ba500ee27385f549e5b4027dcb44f68016b2/app/views/phlex/phorm/field.rb#L11-L39